src/main/frontend/modules/shortcut/config.cljs

```diff
@@ -71,6 +71,78 @@
+   
+   :whiteboard/portal            {:binding "3"
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "logseq-portal")}
+
+   :whiteboard/pencil            {:binding ["4" "d"]
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "pencil")}
+
+   :whiteboard/highlighter       {:binding ["5" "h"]
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "highlighter")}
+   
+   :whiteboard/eraser            {:binding ["6" "e"]
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "erase")}
+   
+   :whiteboard/connector         {:binding ["7" "c"]
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "line")}
+   
+   :whiteboard/text              {:binding ["8" "t"]
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "text")}
+
+   :whiteboard/rectangle         {:binding ["9" "r"]
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "box")}
+
+   :whiteboard/ellipse           {:binding "o"
+                                  :fn      #(.selectTool ^js (state/active-tldraw-app) "ellipse")}
+
+   :whiteboard/reset-zoom        {:binding "shift+0"
+                                  :fn      #(.resetZoom (.-api ^js (state/active-tldraw-app)))}
+
+   :whiteboard/zoom-to-fit       {:binding "shift+1"
+                                  :fn      #(.zoomToFit (.-api ^js (state/active-tldraw-app)))}
+
+   :whiteboard/zoom-to-selection {:binding "shift+2"
+                                  :fn      #(.zoomToSelection (.-api ^js (state/active-tldraw-app)))}
+
+   :whiteboard/zoom-out          {:binding "shift+dash"
+                                  :fn      #(.zoomOut (.-api ^js (state/active-tldraw-app)) false)}
+
+   :whiteboard/zoom-in           {:binding "shift+="
+                                  :fn      #(.zoomIn (.-api ^js (state/active-tldraw-app)) false)}
+
+   :whiteboard/send-backward     {:binding "["
+                                  :fn      #(.sendBackward ^js (state/active-tldraw-app))}
+
+   :whiteboard/send-to-back      {:binding "shift+["
+                                  :fn      #(.sendToBack ^js (state/active-tldraw-app))}
+
+   :whiteboard/bring-forward     {:binding "]"
+                                  :fn      #(.bringForward ^js (state/active-tldraw-app))}
+
+   :whiteboard/bring-to-front    {:binding "shift+]"
+                                  :fn      #(.bringToFront ^js (state/active-tldraw-app))}
+
+   :whiteboard/lock              {:binding "mod+l"
+                                  :fn      #(.setLocked ^js (state/active-tldraw-app) true)}
+
+   :whiteboard/unlock            {:binding "mod+shift+l"
+                                  :fn      #(.setLocked ^js (state/active-tldraw-app) false)}
+
+   :whiteboard/group             {:binding "mod+g"
```

Comment:
I'm not sure why our keybinding system didn't catch it but this and the following shortcut conflict with navigating find-in-page results e.g. :go/electron-jump-to-the-next. This shortcut also conflicts with the main keybinding shortcut for the popular plugin - https://github.com/briansunter/logseq-plugin-gpt3-openai

Reply:
`:go/electron-jump-to-the-next` will only be triggered if search in page is active, so grouping is not blocked by it. I would personally remove the secondary search in page shortcuts if I had to. I tested the plugin, and it's also not blocked by the whiteboard shortcut. The only issue is the notification that pops up when you group elements. Using `mod+g` while in searching in a page along with the plugin, is actually a bigger issue. Both actions will be triggered. Also, the fact that most users will probably just use enter to jump to the next search result, makes me want to remove the secondary search shortcuts. Waiting for your opinion on this.

Reply:
> :go/electron-jump-to-the-next will only be triggered if search in page is active, so grouping is not blocked by it

Confirmed. Good point

> Using mod+g while searching in a page along with the plugin, is actually a bigger issue

For some reason this seems like less of an issue since a user can clearly see that they've triggered a modal and should consider remapping one of the actions

> makes removing the secondary search shortcuts more compelling

These are standard secondary search keybindings in the browser and other apps. Doesn't seem obvious to remove them. This feels out of scope for this PR but feel free to do a follow up issue/PR/slack thread about this